hackathon_site/registration/jinja2/registration/signup.html

```diff
@@ -2,11 +2,14 @@
 
 {% block form_title %}Sign up{% endblock %}
 
-{% block form_img %}<img src="{{ static('event/images/landing.svg') }}"  alt="Signup image" />{% endblock %}
+{% block nav_logout_login_links %}{% endblock %}
+
+{% block form_img %}<img src="{{ static('event/images/signup.svg') }}"  alt="Signup image" />{% endblock %}
 
 {% set form_action=url("registration:signup") %}
                
 {% block form_inputs %}
+<div class="row">
```

Comment:
Since you've added this for all form inputs, can you just put it around the form_input block in form_base.html?

Reply:
facts